src/common.h

```diff
@@ -368,12 +368,10 @@ typedef struct session {
 	int glx_event;
 	/// Error base number for X GLX extension.
 	int glx_error;
-	/// Whether X Xinerama extension exists.
-	bool xinerama_exists;
-	/// Xinerama screen regions.
-	region_t *xinerama_scr_regs;
-	/// Number of Xinerama screens.
-	int xinerama_nscrs;
+	/// Number of XRandR monitors.
+	int xrandr_nmons;
```

Comment:
i know the naming convention is confusion, especially when fixed with older code. but for new code, we prefer to use full words instead of abbreviations.

Reply:
addressed.
1) `xrandr_nmons` => `xrandr_nmonitors`;
2) `xrandr_mon_regs` => `xrandr_monitor_regs`;
3) `xrandr_mon` => `xrandr_monitor`.

is that ok?